highlights.md

```diff
@@ -2,9 +2,178 @@
+    you can now get access to the raw `spdlog` logger, which provides greater
+    control and flexibility. A new macro, `gztrace` is also provided, which can
+    be useful in debugging or data collection scenarios. With `spdlog`, you also
+    get the ability to use format strings. Here's a quick example:
+
+    ```c++
+    auto logger = gz::common::Console::Root().RawLoggerPtr();
+    logger->trace("π to 5th decimal = {:.5f}\n", M_PI);
+    ```
+
+- Improvements to the bullet-featherstone physics plugin
+
+  - Many new features and bugfixes were made to the bullet-featherstone plugin
+    in gz-physics, which include support for nested models,
+    off-diagonal inertia, enforcing joint velocity and effort limits,
+    configuring solver iterations, and more. There are also features for
+    performance improvements and physics stability such as auto-deactivation,
+    static object collision optimization, and use of convex hull shapes for
+    convex decomposed meshes.
+
+- [Support for mesh optimization](https://github.com/gazebosim/gz-sim/pull/2417),
+[gzsim#2352](https://github.com/gazebosim/gz-sim/pull/2352),
+[gz-common#585](https://github.com/gazebosim/gz-common/pull/585),
+[sdformat#1382](https://github.com/gazebosim/gz-common/pull/585), and
+[gz-physics#606](https://github.com/gazebosim/gz-physics/pull/606)
+
+  - Added support for mesh optimization on the collision mesh. Users can now
+    specify whether or not to perform mesh optimization on a
+    collision mesh in SDF (requires SDF spec version >= 1.11). Two optimization
+    methods are currently supported: `convex_decomposition` and `convex_hull`.
+    Gazebo uses the open source
+    [V-HACD](https://github.com/kmammou/v-hacd) library to perform
+    convex decomposition to split the mesh into multiple submeshes.
+    Example SDF usage:
+
+    ```xml
+    <collision>
+      <mesh optimization="convex_decomposition">
+        <convex_decomposition>
+          <max_convex_hulls>16</max_convex_hulls>
+          <voxel_resolution>400000</voxel_resolution>
+        </convex_decomposition>
+        <uri>/path/to/mesh.dae</uri>
+      </mesh>
+    </collision>
+    ```
+
+- [Support specifying plugins in SDF files without overriding default
+  plugins](https://github.com/gazebosim/gz-sim/pull/2497) and [gz-gui#631](https://github.com/gazebosim/gz-gui/pull/631)
+
+  - In prior Gazebo versions, if a user specified a server plugin at the
+    world level, it would override all default plugins. This required
+    users to have in-depth knowledge and specify every plugin necessary
+    for a simulation to run correctly. Omitting critical plugins would
+    lead to unexpected simulation behavior, causing confusion,
+    particularly for new Gazebo users.
+
+  - Gazebo Ionic addresses this issue by loading default plugins even when
+    users specify additional plugins. Gazebo Ionic also introduces
+    `<gz:policy>` settings to revert this behavior to how it functioned in
```

Comment:
we should either add a snippet including `<gz:policy>` or provide a link to documentation about it

Reply:
Added links in https://github.com/gazebosim/gz-ionic/pull/4/commits/5208be50493cdce8db0b73b19eb6a5f1e409f833